CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -1943,10 +1944,14 @@ public static function submit($params) {
     $form->_values['fee'] = $priceSetFields['fields'];
     $form->_priceSetId = $priceSetID;
     $form->setFormAmountFields($priceSetID);
+    $capabilities = array();
```

Comment:
set up environment for api / tests better - normal forms don't call this so really its for the tests